### PR TITLE
Test PHP 8.2

### PR DESCRIPTION
**Pull request by dereuromark**

### CODE REVIEW

**Comment by dereuromark** [comment]
ping @burzum

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version: ['7.4', '8.0', '8.1']
+        php-version: ['7.4', '8.2']
```
Comment (floriankraemer) [inline .github/workflows/ci.yml:12]: Why did you remove 8.0 and 8.1? They can in theory behave different. As long as they are supported I would like to keep them. Same for 7.4 as long as the library technically works with it it should stay even if the php version is no longer supported.

Comment (dereuromark) [reply .github/workflows/ci.yml:12]: There is no point in also checking the ones in between.
